src/main/java/mate/academy/cinemaproject/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.cinemaproject.security;
+
+import mate.academy.cinemaproject.model.User;
+import mate.academy.cinemaproject.service.UserService;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+    private final UserService userService;
+
+    public CustomUserDetailsService(UserService userService) {
+        this.userService = userService;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
+        User user = userService.findByEmail(email).get();
```

Comment:
```suggestion
        User user = userService.findByEmail(email).orElseThrow...
```
давай тут одразу викинемо UserNotFoundException()

Reply:
хорошо